lib/snowplow-tracker/tracker.rb

```diff
@@ -122,38 +120,58 @@ def build_context(context)
 
     # Tracking methods
 
-    # Attaches all the fields in @standard_nv_pairs to the request
+    # Attaches all the fields in @settings to the request
     # Only attaches the context vendor if the event has a custom context
```

Comment:
is this true?

Reply:
I was confused by `emitter.input(payload.context)`. But Payload stores the event details in a hash called context, which is unrelated to the event context (Payload.context might contain the 'co' key, if event context had been given).

Is there any reason for this name or can I change it?

Reply:
Seems strange that the payload is in a property called `context` for sure. `payload.data` is a more popular name across the estate. `co` is the non-base64 equivalent to `cx` where the entity/context array ends up, I don't think there's much relation to this name so I think changing it is less confusing.